src/server/multi_svr.c

```diff
@@ -279,3 +279,30 @@ init_msi()
 
 	return 0;
 }
+
+/**
+ * @brief
+ * 	Used to get serverer_instance_id which is of the form server_instance_name:server_instance_port
+ * 
+ * @return char *
+ * @return NULL - failure
+ * @retval !NULL - pointer to server_instance_id
+ */
+char *
+get_svr_inst_id(void)
+{
+	char svr_inst_name[PBS_MAXHOSTNAME + 1];
+	unsigned int svr_inst_port;
+	char *svr_inst_id = NULL;
+
+
+	if (gethostname(svr_inst_name, PBS_MAXHOSTNAME) == 0)
+        	get_fullhostname(svr_inst_name, svr_inst_name, PBS_MAXHOSTNAME);
```

Comment:
No need to handle possible failure of get_fullhostname()? It can return -1 right?

Reply:
In that case, we still retain the original value